node-lib/src/regtest_options.rs

```diff
@@ -61,4 +62,8 @@ pub struct ChainConfigOptions {
     /// PoS NetUpgrade override after Genesis
     #[clap(long)]
     pub chain_pos_netupgrades: Option<bool>,
+
+    /// PoS Genesis staking settings
+    #[clap(long)]
+    pub chain_genesis_staking_settings: Option<GenesisStakingSettings>,
```

Comment:
Not quite sure about it but you may be able to set something like `default_value = ""` in the `clap` attribute and get rid of the `Option` wrapper.

Reply:
Yep, updated with `default_value_t`.